massgov/mayflower-patternlab
# DP-1370 Radio buttons and checkboxes more visible in feedback forms

Accessibility fix to make the radio buttons and check boxes more visible on feedback form. The border color for these were changed to #757575 per Sarena and Harlan request in the comments. These checkbox and radio buttons borders were changed from border-color: $c-bd-input; to border-color: #757575;

JIRA ticket: https://jira.state.ma.us/browse/DP-1370

To test:
Compare the radio button with the current pilot.mass.gov/topics/health-social-services feedback form.
Inspect the radio button and change the .ma__input-radio border color to #757575
To compare the check box go to http://localhost:3000/?p=pages-pilot-transition vs current site https://pilot.mass.gov/leaving-pilot?continueURL=http://www.mass.gov/dor/child-support/court-information/paternity-establishment/
